.gitignore

```diff
@@ -14,9 +14,10 @@ target/
 tmp/
 gen-external-apklibs/
 out/
+some/
```

Comment:
We should really figure out who creates this directory.

Reply:
HTMLFormatterTest is the culprit.

Reply:
I'll make a seperate PR to fix that.